packages/notifier/test/iterable-testing-tools.js

```diff
@@ -6,6 +6,8 @@ import { observeIteration, observeIterator } from '../src/index.js';
 import '@agoric/marshal/exported.js';
 import '../src/types.js';
 
+/** @typedef {import('ava').Assertions} Assertions */
```

Comment:
Ah, I see. I thought it might be something from the assert package and was confused. I should have commented on it in my earlier review.